manager/src/lib.rs

```diff
@@ -1269,7 +1286,7 @@ mod tests {
         let slot = contract.get_slot_id(Some(1_000_000_000_000));
 
         // ensure even if we pass in a HUGE number, it can only be scheduled UP to the max pre-defined block settings
-        assert_eq!(slot, 1_052_200_980);
+        assert_eq!(slot, 1_000_052_200_980);
```

Comment:
@mikedotexe this looks odd -- Pretty sure the idea here was to only allow a max block so we dont schedule too far in advance, and check with this test